spec/unit/application_spec.rb

```diff
@@ -473,6 +473,33 @@ def configure_proxy_environment_variables_stubs
     end
   end
 
+  describe 'run_chef_client' do
+    before do
+      @app = Chef::Application.new
+      @app.stub(:fork_chef_client)
+      @app.stub(:run_with_graceful_exit_option)
+    end
```

Comment:
are they documented elsewhere? I just went with the style of the rest of the file. Should I update the whole file?